Common/Vulkan/VulkanContext.cpp

```diff
@@ -146,10 +146,14 @@ VkResult VulkanContext::CreateInstance(const CreateInfo &info) {
 		}
 	}
 
+	// Temporary hack for libretro. For some reason, when we try to load the functions from this extension,
+	// we get null pointers when running libretro. Quite strange.
```

Comment:
`vkCreateInstance` is sort of stubbed with libretro since the frontend already created it, which means `instance_extensions_enabled_` is currently being ignored. 
I can't think of an easy way to solve this other than having the frontend always load the extension when available, the current libretro_vulkan api lacks the ability to allow the core to request specific instance extensions.

Reply:
Okay, that does make sense at least. We'll just have to deal with that then, there's no urgent need for that extension anyway.